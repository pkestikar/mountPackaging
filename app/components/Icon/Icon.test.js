import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

describe('Icon component', () => {

    test('if the icon component renders without errors', () => {

    });

});
