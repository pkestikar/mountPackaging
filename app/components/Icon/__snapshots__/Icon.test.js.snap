// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon component if the icon component renders without errors 1`] = `
<ReactSVG
  afterInjection={[Function]}
  beforeInjection={[Function]}
  className="mp--icon"
  evalScripts="never"
  fallback={null}
  loading={null}
  onClick={[Function]}
  renumerateIRIElements={true}
  src="undefined"
  wrapper="span"
/>
`;

exports[`Icon component if the third party react-svg module renders without errors 1`] = `
<span
  className="mp--icon"
  onClick={[Function]}
/>
`;
